ipam/address/address.go

```diff
@@ -98,3 +98,10 @@ func Min(a, b Offset) Offset {
 	}
 	return a
 }
+
+func (addr Address) Reverse() Address {
+	return ((addr >> 24) & 0xff) | // move byte 3 to byte 0
+		((addr << 8) & 0xff0000) | // move byte 1 to byte 2
+		((addr >> 8) & 0xff00) | // move byte 2 to byte 1
+		((addr << 24) & 0xff000000) // byte 0 to byte 3
+}
```

Comment:
It's odd for `nameserver` to import something from IPAM. Let's move `address`, including the above function, into `net`. In a separate PR.